stylelint-config.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-standard",
+  "plugins": ["stylelint-scss", "stylelint-order"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true
+  }
```

There are some pure CSS files around:

```suggestion
  "overrides": [
    {
      "files": ["*.scss", "**/*.scss"],
      "rules": {
        "at-rule-no-unknown": null,
        "scss/at-rule-no-unknown": true
      }
    }
  }
```

---

Is this just so we have a place to put scss specific rules? Because without the override it still successfully lints all CSS/SCSS files if you can `yarn stylelint "**/*.{css,scss,sass}"`

---

It’s to run `scss/at-rule-no-unknown` only on `.scss` files. With current setup you can use SCSS–specific at-rules in `.css` files.

---

Gotcha 👍